scipy/special/add_newdocs.py

```diff
@@ -1310,9 +1310,16 @@ def add_newdoc(place, name, doc):
     the amplitude of `u`.
 
     Computation is by means of the arithmetic-geometric mean algorithm,
-    except when `m` is within 1e-9 of 0 or 1.  In the latter case with `m`
-    close to 1, the approximation applies only for `phi < pi/2`.
-
+    except when `m` is within 1e-9 of 0 or 1.  
+    
+    In the latter case, when m is near 1, the functions `sn`, `cn`, and `dn`
+    are found using an approximation for `phi` given by 16.15.4 in [2]. 
+    As this approximation is only valid when `phi < pi/2` (i.e.,
+    when `u<K`), the computation only uses 16.15.4 to find the "principal" 
+    part of `phi` or the remainder `dphi=phi%(pi/2)`. The returned value 
```

Comment:
double-backticks on code for it to render properly

Reply:
Which parts? Do you just mean the dphi=phi%(pi/2) part? Or everywhere I've used one set of backticks (e.g., should `phi` just be italicized (one pair of `) or as code (two pairs of `)?)